ApiContract/request.json5

```diff
@@ -0,0 +1,41 @@
+//POST /api/advertisement
+
+//Request object
+{
+	//[Mandatory] id of the advertisement
+	"id" : "P123",
+	//[Mandatory] title of the advertisement
+	"name" : "Watch" ,
+	//[Mandatory] description of the advertisement
+	"description" : "something",
+	//[Mandatory] images of the product in the advertisement (Maximum files - 5)
+	"heroImageUrl" : "www.example.com/imageName",
+	//[Mandatory] 
+	"price" : 
+	{
+		// price of the product in the advertisement (in Rupees)
+		"amount" : "300",
+		// Boolean flag specifying whether price is negotiable or not
+		"isPriceNegotiable" : "true"
+		
+	},
+	//[Mandatory] category of the product in the advertisement
+	"category" : "Accessories",
+
+	//[Optional] images of the product
+	"imageUrls" : [
+		{
+			"url" : "www.example.com/imageName",
+			"order" : "1"
+		},
+		{
+			"url" : "www.example.com/imageName",
+			"order" : "2"
+		}],
+	//[Optional] date when the original product was purchased
+	"purchaseDate" : "yyyy'-'MM'-'dd'T'HH':'mm':'ss'.'fffffffz",
```

Comment:
this is a general format of date which should be in comment, whereas in value of a contract should be "2019-09-01T01:05:99.33"